### PR TITLE
doc: update readme

### PR DESCRIPTION
**Pull request by samen93**

### CODE REVIEW

**Review by soerenmartius — DISMISSED** [review]
lgtm!

**Review by mariux — APPROVED** [review]
lgtm

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 [![Google Provider Version][badge-tf-gcp]][releases-google-provider]
 [![Join Slack][badge-slack]][slack]
 
-# terraform-google-cloud-function
+# terraform-google-artifact-registry-repository
```
Comment (mariux) [inline README.md:7]: ?? this is cloud-function...

File: README.md [hunk 2]
```diff
@@ -4,9 +4,9 @@
...
 
-A [Terraform](https://www.terraform.io) module to create a [Google Cloud Function](https://cloud.google.com/functions/docs) on [Google Cloud Services (GCP)](https://cloud.google.com/).
+A [Terraform] module for [Google Cloud Platform (GCP)][gcp].
```
Comment (mariux) [inline README.md:9]: still cloud function

**Review by mariux — CHANGES_REQUESTED** [review]
@samen93 this PR does not make sense to me...

**Review by soerenmartius — APPROVED** [review]
lgtm!

**Review by mariux — APPROVED** [review]

**Pull request merged**